rootfs/etc/nginx/lua/balancer/chashsubset.lua

```diff
@@ -0,0 +1,92 @@
+-- Consistent hashing to a subset of nodes. Instead of returning the same node
+-- always, we return the same subset always.
+
+local resty_chash = require("resty.chash")
+local util = require("util")
+
+local _M = { name = "chashsubset" }
+
+local function build_nodeset_map(backend)
+  local nodes = {}
+  local nodeset_map = {}
+  local node_sets = {}
```

Comment:
Annotation refers to this as `subset` and also the Go code. Can you keep the naming consistent here as well and call this something like `subsets`?